planning/planning_test_utils/include/planning_test_utils/planning_test_utils.hpp

```diff
@@ -79,20 +79,248 @@ using tier4_autoware_utils::createQuaternionFromRPY;
+
+/**
+ * @brief Creates a LaneletSegment with the specified ID.
+ *
+ * Initializes a LaneletSegment containing a single LaneletPrimitive with the
+ * given ID and a primitive type of "lane".
+ *
+ * @param id The ID for the LaneletPrimitive and preferred primitive.
+ * @return A LaneletSegment with the specified ID.
+ */
+LaneletSegment createLaneletSegment(int id);
+
+/**
+ * @brief Loads a Lanelet map from a specified file.
+ *
+ * This function loads a Lanelet2 map using the given filename. It uses the MGRS
+ * projector and checks for any errors during the loading process. If errors
+ * are found, they are logged, and the function returns nullptr.
+ *
+ * @param lanelet2_filename The filename of the Lanelet2 map to load.
+ * @return A LaneletMapPtr to the loaded map, or nullptr if there were errors.
+ */
+lanelet::LaneletMapPtr loadMap(const std::string & lanelet2_filename);
+
+/**
+ * @brief Converts a Lanelet map to a HADMapBin message.
+ *
+ * This function converts a given Lanelet map to a HADMapBin message. It also
+ * parses the format and map versions from the specified filename and includes
+ * them in the message. The timestamp for the message is set to the provided time.
+ *
+ * @param map The Lanelet map to convert.
+ * @param lanelet2_filename The filename of the Lanelet2 map, used to parse format and map versions.
+ * @param now The current time to set in the message header.
+ * @return A HADMapBin message containing the converted map data.
+ */
+HADMapBin convertToMapBinMsg(
+  const lanelet::LaneletMapPtr map, const std::string & lanelet2_filename,
+  const rclcpp::Time & now);
+
+/**
+ * @brief Creates a normal Lanelet route with predefined start and goal poses.
+ *
+ * This function initializes a LaneletRoute with a predefined start and goal pose.
+ *
+ * @return A LaneletRoute with the specified start and goal poses.
+ */
+LaneletRoute makeNormalRoute();
+
+/**
+ * @brief Creates an OccupancyGrid message with the specified dimensions and resolution.
+ *
+ * This function initializes an OccupancyGrid message with given width, height, and resolution.
+ * All cells in the grid are initialized to zero.
+ *
+ * @param width The width of the occupancy grid.
+ * @param height The height of the occupancy grid.
+ * @param resolution The resolution of the occupancy grid.
+ * @return An OccupancyGrid message with the specified dimensions and resolution.
+ */
+OccupancyGrid makeCostMapMsg(size_t width = 150, size_t height = 150, double resolution = 0.2);
+
+/**
+ * @brief Creates a HADMapBin message from a Lanelet map file.
+ *
+ * This function loads a Lanelet map from the given file, overwrites the
+ * centerline with the specified resolution, and converts the map to a HADMapBin message.
+ *
+ * @param absolute_path The absolute path to the Lanelet2 map file.
+ * @param center_line_resolution The resolution for the centerline.
+ * @return A HADMapBin message containing the map data.
+ */
+HADMapBin make_map_bin_msg(
```

Comment:
Do you have any distinguish between snake case and camel case?
(I know snake case is recommended for function name though

Reply:
@kyoichi-sugahara 
For old function, I keep it as camelCase to avoid having to change other code. For new function, I used snake_case.